packages/wonder-blocks-form/src/__tests__/__snapshots__/custom-snapshot.test.tsx.snap

```diff
@@ -2,12 +2,14 @@
 
 exports[`CheckboxCore type:default checked:false 1`] = `
 <input
+  aria-checked="false"
```

Comment:
**suggestion:** that makes me realize that we should add a new variant to cover the `indeterminate` case.

This test can be included here: https://github.com/Khan/wonder-blocks/blob/c2ec490259d77ec44dcd6b868d050314c93d72e9/packages/wonder-blocks-form/src/__tests__/custom-snapshot.test.tsx#L9